cmd/cstor-pool-mgmt/controller/common/common.go

```diff
@@ -126,8 +127,19 @@ const (
 	QOpModify  QueueOperation = "modify"
 )
 
-// IsImported is channel to block cvr until certain pool import operations are over.
-var IsImported chan bool
+// namespace defines kubernetes namespace specified for cvr.
+type namespace string
+
+// Different types of k8s namespaces.
+const (
+	defaultNameSpace namespace = "default"
+)
+
+// Mux is mutex variable to block cvr until certain pool operations are complete.
+var Mux *sync.Mutex
```

Comment:
Instead of having Mux/IsImported properties file scoped, can we instead have a structure with these properties embedded? That would give better control and avoid any accidental changes.